vic.api.v0.yaml

```
@@ -2009,6 +2011,28 @@ components:
         anyOf:
           - $ref: '#/components/schemas/InvoiceState'
           - enum: [ALL, POSTED]
+    InvoiceMarkedAsFilter:
+      name: markedAs
+      in: query
+      description: |
+        selects the marked as of invoices which are to be searched
+        (defaults to `ALL`)
+      required: false
+      schema:
+        anyOf:
+          - $ref: '#/components/schemas/InvoiceMarkedAs'
+          - enum: [ALL, PAID]
```

`PAID` is already in `InvoiceMarkedAs`

```suggestion
          - enum: [ALL]
```